### PR TITLE
fix(codestyle): fix PSR2 incompatible changes and update packages

### PR DESCRIPTION
**Pull request by victorvassilev**
## Description

fix sonarqube errors for 19.04 branch

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [x] 19.04.x
- [ ] 19.10.x (master)

### CODE REVIEW

File: src/CentreonRemote/Domain/Service/TaskService.php [hunk 1]
```diff
@@ -148,7 +148,8 @@ public function getRemoteStatusByParent(int $parentId, string $serverIp, string
             $result = unserialize($remoteDataResult[0]->params);
 
             $httpMethod = $result['params']['http_method'];
-            $httpPort = $result['params']['http_port'];;
+            $httpPort = $result['params']['http_port'];
+            ;
```
Comment (sc979) [inline src/CentreonRemote/Domain/Service/TaskService.php]: is this line needed ?

**Comment by sc979** [comment]
Please could you correct these one two :
image

File: src/CentreonRemote/Domain/Service/TaskService.php (hunk 1 above)
Comment (victorvassilev) [reply src/CentreonRemote/Domain/Service/TaskService.php]: I used php cs fixer, I guess it moved it down instead of delete the extra semicolon :)

**Comment by victorvassilev** [comment]
> Please could you correct these one two :
> image

this was already fixed as part of the push

**Comment by sc979** [comment]
these are the current coding style issues reported with sonar after your "menu rework" merge.
Please check the CI reports

File: src/CentreonRemote/Domain/Service/TaskService.php (hunk 1 above)
Comment (vhr) [inline src/CentreonRemote/Domain/Service/TaskService.php]:
```suggestion
```

no needs in the second semicolon

Comment (victorvassilev) [reply src/CentreonRemote/Domain/Service/TaskService.php]: removed

**Comment by lgcosta** [comment]
@victorvassilev taking advantage of having several changes in the files, I think it is also interesting to update the copyright year in the license headers.

**Pull request merged**